From: Daniel Wagner <dwagner@suse.de>
Date: Thu Jun 15 11:48:53 AM PDT 2023
Subject: tracecmd: Add explicit meson thread dependency  
Patch-mainline: mailed to mailing list
References: none
Signed-off-by: Tony Jones <tonyj@suse.de>

Older version of meson do not add automatically the pthread and dl
dependency. Thus add it explicitly to the build.

Signed-off-by: Daniel Wagner <dwagner@suse.de>
---
 meson.build | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/meson.build b/meson.build
index 906ac8168f4f..fbdf016f67a7 100644
--- a/meson.build
+++ b/meson.build
@@ -27,6 +27,9 @@ conf = configuration_data()
 libtraceevent_dep = dependency('libtraceevent', version: '>= 1.5.0', required: true)
 libtracefs_dep = dependency('libtracefs', version: '>= 1.6.0', required: true)
 
+threads_dep = dependency('threads', required: true)
+dl_dep = cc.find_library('dl', required : false)
+
 zlib_dep = dependency('zlib', required: false)
 conf.set('HAVE_ZLIB', zlib_dep.found(), description: 'Is zlib avialable?')
 
-- 
2.40.1

